js/webpack/app.js

```diff
@@ -67,7 +67,7 @@ module.exports = {
       },
       {
         test: /\.js$/,
-        include: /node_modules\/(material-chip-input|ethereumjs-tx)/,
```

Comment:
I have something in the back of my head to look at the new Babel uglifier, thanks for the reminder in logging that - just need to see if there are any benefits/drawbacks.